src/libraries/Common/src/Interop/Windows/WinSock/Interop.WSASendTo.cs

```diff
@@ -18,7 +18,7 @@ internal static partial class Winsock
             int bufferCount,
             out int bytesTransferred,
             SocketFlags socketFlags,
-            IntPtr socketAddress,
+            Span<byte> socketAddress,
```

Comment:
You could make this ReadOnlySpan to highlight that WSASendTo is not mutating the data.